terraform/modules/services/sat/secrets.tf

```diff
@@ -0,0 +1,11 @@
+# Read in secrets for API for EUMETSAT API
+
+data "aws_secretsmanager_secret" "sat-api" {
+  name = "development/consumer/sat"
+  #  arn = "arn:aws:secretsmanager:eu-west-2::secret:development/consumer/nwp"
```

Comment:
remove if not needed

Reply:
This should be needed actually, I think, as I need to read in the EUMETSAT API keys

Reply:
sorry i meant the comment  `# arn`